<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <widget class="QPushButton" name="btn_register">
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </layout>
  </widget>
 </widget>
</ui>
